Packs/CrowdStrikeHost/Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -1056,6 +1130,36 @@ script:
       description: Technique of the behavior
       type: string
     description: Fetch details of a Crowdstrike Detection using the detection ID
+  - name: cs-threatgraph-summary
+    arguments:
+    - name: ctg_id
+      required: true
+      description: Crowd Strike threar grpah ID.
+      isArray: true
+    outputs:
+    - contextPath: CrowdStrikeHost.ThreatGraphSummary
```

Comment:
context outputs in all three commands should be reviewed. this is just dumping the json. do we want to expose more data? see other commands.